README.md

```diff
@@ -25,7 +25,22 @@ Unlike other observability backends, it has a simple and easy-to-manage architec
 with just two components: the Promscale Connector and the Promscale Database (PostgreSQL with the
 TimescaleDB and Promscale extensions).
 
-Key features:
+## Quick Start
+
+**Try it out now** with our demo environment you can deploy on your laptop in five minutes with Docker.
+
+```bash
+git clone git@github.com:timescale/promscale.git
```

Comment:
nit: It is always better to use git over http instead of git over git proto to avoid potential port blocking in a larger orgs.
```suggestion
git clone https://github.com/timescale/promscale.git
```